tests/account_test.js

```diff
@@ -2,10 +2,20 @@ const r_ship = Math.floor(Math.random() * 5) + 1
 
 Feature('Guestbook');
     
-Scenario('verify username and add entry to guestbook',  ({ I }) => {
+Scenario('verify username',  ({ I }) => {
```

Comment:
Scenario names should start with `I can ...`, so maybe put `I can verify the heading has the correct obit name`. Just an example.

Reply:
I don't always name scenarios that way, I think it depends on the test, and personal preference.  The Scenario should give a good summary of its purpose though.

Reply:
Updates made